docs/building-rpm-ostree-and-autosd.md

```diff
@@ -0,0 +1,171 @@
+<!-- markdownlint-disable-file MD013 MD033 -->
+# Building rpm-ostree in autosd image from scratch
+
+The AutoSD is an upstream repository for Red Hat In-Vehicle OS, much like CentOS Stream is to RHEL. AutoSD is based on CentOS Stream with a few divergences. If your base system is not a CentOS, you can use a container to generate a compatible CentOS Stream 9 RPM as showed below, otherwise just skip the step and use your raw system.
+
+## Building rpm-ostree rpm on CentOS Stream9
+
+Create a **ContainerFile** with the content below.
+
+```bash
+FROM centos:stream9
+
+RUN dnf update -y
+RUN dnf install -y 'dnf-command(config-manager)' epel-release
+RUN dnf config-manager --set-enabled crb
+RUN dnf config-manager --add-repo https://buildlogs.centos.org/9-stream/automotive/aarch64/packages-main/
+RUN dnf config-manager --add-repo https://buildlogs.centos.org/9-stream/autosd/aarch64/packages-main/
+RUN if [ $(uname -m) == "aarch64" ]; then dnf install -y --nogpgcheck autosig-u-boot; fi
```

Comment:
Why is this here btw?  Is it really needed?

Reply:
This Containerfile is based on a hacky one, I've used to build all sorts of rpms:

https://github.com/ericcurtin/staging/blob/master/Mockfile

I never really intended for this to be used by anyone other than me, so a lot of this is overkill in the rpm-ostree context.

This line is not needed to build rpm-ostree rpms.

Reply:
Is the other auto-specific stuff needed too?  I doubt it.

Reply:
It's not needed, it would probably be better if it was just a generic CentOS Stream 9 container.

I'd likely just check in this Containerfile, rather than document it also if we intend on people using it.